Task/Tasks/TSKSelectorTask.h

```diff
@@ -77,7 +93,7 @@
      May not be NULL.
  @result A newly initialized TSKSelectorTask instance with the specified name, target, and action.
  */
-- (instancetype)initWithName:(NSString *)name target:(id)target selector:(SEL)selector;
+- (instancetype)initWithName:(nullable NSString *)name target:(id)target selector:(SEL)selector;
```

Comment:
I think the property declaration for `target` needs a `nullable` attribute. (I can't add a comment on that line b/c github is stupid)